test/test_tasks.py

```diff
@@ -410,6 +410,28 @@ def test_multiple_activities_no_target_urls(self):
     self.assert_equals('complete', resp.status)
     self.assertIsNone(resp.urls_to_activity)
 
+  def test_mentions_only_go_to_posts_and_comments(self):
```

Comment:
you mean the method name? yeah, true. will fix. i think the test itself is right though...?